backend/app/history/chat_history.py

```diff
@@ -0,0 +1,84 @@
+"""
+This file contains all the logic for communicating with the database with regard to chat history messages.
+Chat messages are managed using a session key and set to expire after some time if not used.
+
+Author: Ruben Vandamme
+Date: 05 November 2024
+Usage:
+    - add_message           : save a message
+    - get_chat_history      : get messages using a session key
+    - (refresh_session)     : refresh messages TTL of a session
+    - find_session_id       : check if a session id is present
+    - generate_session_id   : generate an unique session id
+"""
+
+from pymongo import MongoClient
+from pymongo import ASCENDING
+import datetime
+from pydantic import BaseModel
+from typing import List
+from enum import StrEnum
+import secrets
+import string
+
+# setup connection
+client = MongoClient('mongodb://localhost:27017/')
```

Comment:
Could you make the `IP:port` a ENV variable with some default? 
Also note "localhost" is usually a bad practice since it might resolve into IPv4 or IPv6 and sometimes might break for some people.